sdk/rms_sdk/Profile/SConscript

```diff
@@ -0,0 +1,12 @@
+from build_config import *
+
+Import('rms_env')
+
+profile_env = rms_env.Clone()
+
+src_files = Glob('*.cpp')
+profile_env.Append(CPPPATH='#sdk/rms_sdk/Logger')
```

Comment:
Where you able to figure out if these CPPATH were necessary?

Reply:
Yes. These are necessary